firebase.json

```diff
@@ -8,6 +8,10 @@
         "source": "/sitemap.xml",
         "function": "sitemap"
       },
+      {
+        "source": "/catalog/**",
+        "function": "catalog"
+      },
```

Comment:
According to this rewrite definition, all `/catalog/**` requests are sent to the Firebase Functions.